go/oasis-node/cmd/debug/txsource/workload/governance.go

```diff
@@ -204,7 +204,7 @@ func (g *governanceWorkload) submitCancelUpgradeProposal(descriptor *upgrade.Des
 	}
 	var proposal *governance.Proposal
 	for _, p := range ps {
```

Comment:
Could we update this txsource governance workload to also submit some fuzzed new proposal types?

Reply:
Added randomly generated change parameters proposals.